x-pack/plugins/infra/public/pages/metrics/inventory_view/components/waffle/conditional_tooltip.tsx

```diff
@@ -18,6 +20,8 @@ import { InfraWaffleMapNode, InfraWaffleMapOptions } from '../../../../../lib/li
 import { useSnapshot } from '../../hooks/use_snaphot';
 import { createInventoryMetricFormatter } from '../../lib/create_inventory_metric_formatter';
 import { SNAPSHOT_METRIC_TRANSLATIONS } from '../../../../../../common/inventory_models/intl_strings';
+import { useWaffleOptionsContext } from '../../hooks/use_waffle_options';
+import { createFormatterForMetric } from '../../../metrics_explorer/components/helpers/create_formatter_for_metric';
```

Comment:
I don't think the `formatter` prop of this component is being used as we import one. Perhaps it should be removed.